data/styles/vehicle/include/priorities_4_overlays.prio.txt

```diff
@@ -663,6 +662,11 @@ railway-subway_entrance-yokohama                    # icon z16- (also has captio
 
 amenity-parking-no-access                           # icon z15- (also has caption(optional) z16-, area z15-)
 amenity-parking-private                             # icon z17- (also has caption(optional) z17-, area z15-)
+amenity-parking-private-street_side                 # icon z17- (also has caption(optional) z17-, area z15-)
```

Comment:
make it less visible than a public counterpart?

Reply:
I added it to z17- because `amenity-parking-private` shows in z17. Should it be lower?

Reply:
do you agree to it?

Reply:
>Actually now sure how come amenity-parking is z15- in vehicle just like in the main :) IMO it should be either lower zoom in vehicle or higher in the main :)

`amenity-parking` is z16- in main style and z15- in vehicle style

Reply:
I have set the following zooms for icons:

Main style:
 ```
amenity-parking z16-
amenity-parking-street_side z18-
amenity-parking-lane z18-
amenity-parking-private z18-
amenity-parking-private-street_side z18-
amenity-parking-private-lane z19-
```

Vehicle style:
```
amenity-parking z15-
amenity-parking-street_side z17-
amenity-parking-lane z17-
amenity-parking-private z17-
amenity-parking-private-street_side z17-
amenity-parking-private-lane z18-
```

Do you think it's good?

Reply:
> 
> `amenity-parking` is z16- in main style and z15- in vehicle style

Sorry I meant area fill is visible from z15- in both styles indeed, an icon is z16- in the main style indeed

Reply:
I think its good, thanks for the changes.
We might adjust area fills visibility later if needed.